### PR TITLE
Use the official boxes from VagrantCloud when available

### PR DESCRIPTION
**Pull request by dcermak**

### CODE REVIEW

File: seslib/constant.py [hunk 1]
```diff
@@ -63,9 +63,8 @@ class Constant():
         'leap-15.1': 'https://download.opensuse.org/repositories/Virtualization:/'
                      'Appliances:/Images:/openSUSE-Leap-15.1/images/'
                      'Leap-15.1.x86_64-libvirt.box',
-        'tumbleweed': 'https://download.opensuse.org/repositories/Virtualization:/'
-                      'Appliances:/Images:/openSUSE-Tumbleweed/openSUSE_Tumbleweed/'
-                      'Tumbleweed.x86_64-libvirt.box',
+        'tumbleweed': 'opensuse/Tumbleweed.x86_64',
+        'leap-15.2': 'opensuse/Leap-15.2.x86_64',
```
Comment (smithfarm) [inline seslib/constant.py:67]: this causes a linter error:

```
************* Module seslib.constant

seslib/constant.py:62:21: W0109: Duplicate key 'leap-15.2' in dictionary (duplicate-key)
```

(I guess you want to remove the existing `leap-15.2` key/value pair)

Comment (dcermak) [reply seslib/constant.py]: Yep, I completely missed that key. Should be fixed now.

**Comment by smithfarm** [comment]
```
==> box: Loading metadata for box 'opensuse/Leap-15.2.x86_64'
    box: URL: https://vagrantcloud.com/opensuse/Leap-15.2.x86_64
Command '['vagrant', 'box', 'add', '--provider', 'libvirt', '--name', 'leap-15.2', 'opensuse/Leap-15.2.
x86_64']' failed: ret=1 stderr:
The box you're adding has a name different from the name you
requested. For boxes with metadata, you cannot override the name.
If you're adding a box using `vagrant box add`, don't specify
the `--name` parameter. If the box is being added via a Vagrantfile,
change the `config.vm.box` value to match the name below.
```

**Comment by smithfarm** [comment]
@dcermak Any chance you could remove the hard-coded name from the box?

**Comment by dcermak** [comment]
> @dcermak Any chance you could remove the hard-coded name from the box?

Unfortunately not really, the box name comes from vagrant cloud and is afaik set automatically by the vagrant cloud API.

**Comment by smithfarm** [comment]
> Unfortunately not really, the box name comes from vagrant cloud and is afaik set automatically by the vagrant cloud API.

OK, I'm working on a patchset to map these hard-coded Box names to sesdev's human-friendsly box/os names.

**Comment by smithfarm** [comment]
This is not so easy, because users will already have a local box called `leap-15.2`. If I make that name suddenly point to a different box, breakage will ensue. But I think I can make it work.

**Comment by smithfarm** [comment]
Integrated this commit into #476

**Pull request closed without merging**